src/app/zones/views/ZoneDetailView.vue

```diff
@@ -12,7 +12,7 @@
             name: 'zone-cp-list-view',
```

Comment:
That’s correct. The ZoneListView and ZoneDetailView are technically named incorrectly. I wouldn’t worry about doing that now though (unless you want to). To me, that causes only little confusion.

Reply:
Yes definitely! Now we have a concept of zones with separate ZoneCPs I think this should be renamed soon.

Quick FYI, there is a load of renaming/route config noodling I would like to do as a lot of it isn't consistent etc. I was half waiting for a better time to do this, when all the feature/fixing work around here had settled down and we have more e2e test coverage around it.

I suppose the thing is "when is the best time to go through and address all the inconsistent naming", I would guess it would be pretty soon.